demos/ota/aws_iot_ota_update_demo.c

```diff
@@ -48,242 +50,257 @@
 #include "task.h"
 #include "semphr.h"
 
-/* Demo network handling */
-#include "aws_iot_demo_network.h"
-
 /* Required to get the broker address and port. */
 #include "aws_clientcredential.h"
 
 /* FreeRTOS OTA agent includes. */
 #include "aws_iot_ota_agent.h"
 
-#include "iot_network_manager_private.h"
-
 /* Required for demo task stack and priority */
 #include "aws_demo_config.h"
 #include "aws_application_version.h"
 
-static void App_OTACompleteCallback( OTA_JobEvent_t eEvent );
-
-static BaseType_t prxCreateNetworkConnection( void );
-static IotNetworkError_t prvNetworkDisconnectCallback( void * pvContext );
-static void prvNetworkStateChangeCallback( uint32_t ulNetworkType,
-                                           AwsIotNetworkState_t xNetworkState,
-                                           void * pvContext );
-
-/*-----------------------------------------------------------*/
-
-#define otaDemoCONN_TIMEOUT_MS            ( 2000UL )
-
-#define otaDemoCONN_RETRY_INTERVAL_MS     ( 5000 )
-
-#define otaDemoCONN_RETRY_LIMIT           ( 100 )
-
-#define otaDemoKEEPALIVE_SECONDS          ( 1200 )
-
-#define myappONE_SECOND_DELAY_IN_TICKS    pdMS_TO_TICKS( 1000UL )
+/**
+ * @brief Timeout for MQTT connection, if the MQTT connection is not establihsed within
```

Comment:
spelling: _established_

Reply:
Fixing this, Thank you for pointing it out.